spec/requests/api/graphql/queries/submission_spec.rb

```diff
@@ -102,5 +102,49 @@
         )
       end
     end
+
+    context 'including offers' do
+      let(:partner) { Fabricate :partner }
+      let(:partner_submission) do
+        Fabricate :partner_submission, partner: partner, submission: submission
+      end
+      let!(:offer) { Fabricate :offer, partner_submission: partner_submission }
+
+      let!(:another_offer) do
+        partner_submission =
+          Fabricate :partner_submission, submission: submission
+        Fabricate :offer, partner_submission: partner_submission
+      end
+
+      let(:query) do
+        <<-GRAPHQL
+        query {
+          submission(#{query_inputs}) {
+            id,
+            artistId,
+            title
+            offers(gravityPartnerId: "#{partner.gravity_partner_id}") {
+              id
+              state
+              commissionPercent
```

Comment:
This might be an opinion, but there are several fields in this query that don't appear in the test assertions. If they're not worth asserting in the results, IMO they shouldn't be in the query - they have the potential to throw people off from what this test is actually testing. 

(Shameless plug, I wrote a blog post about this https://www.stevenhicks.me/blog/2018/01/chekhovs-gun-and-better-unit-tests/)

Reply:
> If a line of setup code isn’t important to describe a specific test, then I don’t want it to distract me.

I really like that line! With this in mind going to take another pass at this file and see if I can adhere to this principle more closely! 👍

Reply:
Addressed with 3c77374 and fixed a bug along the way!